### ISSUE TITLE
Dotfiles in nested directories aren't renamed

### ISSUE THREAD

**Issue by insin** [yoshuawuyts/copy-template-dir#2]
I have a template directory with a `tests/_.eslintrc` file which isn't getting renamed by `reDot`.

This tweak seems to do the trick:

``` js
function reDot (filepath) {
  const parts = filepath.split(path.sep)
  const filename = parts.pop().replace(/^\_./, '.')
  return parts.concat([filename]).join(path.sep)
}
```

**Comment by yoshuawuyts** [comment]
Ohey, that's a good one. PR definitely welcome.

**Comment by insin** [comment]
Fixed in bd32fdf
